src/content/docs/fr/guides/cms/flotiq.mdx

```diff
@@ -0,0 +1,294 @@
+---
+title: Flotiq & Astro
+description: Ajouter du contenu à votre projet Astro en utilisant Flotiq comme CMS
+type: cms
+service: Flotiq
+i18nReady: true
+---
+
+import PackageManagerTabs from '~/components/tabs/PackageManagerTabs.astro';
+import { Steps } from '@astrojs/starlight/components';
+
+[Flotiq](https://flotiq.com) est un CMS headless conçu pour différents frontends, tels que les sites statiques, les applications mobiles et les applications web. Les développeurs et les créateurs de contenu gèrent et diffusent le contenu par le biais d'API REST et GraphQL.
+
+## Intégration avec Astro
+
+Ce guide permet d'utiliser l'API Flotiq headless CMS avec un projet Astro pour afficher du contenu sur votre site web.
```

Comment:
The wording seems a bit strange to me, perhaps we can find something simpler to read:
```suggestion
Ce guide permet d'utiliser l'API CMS headless de Flotiq avec un projet Astro pour afficher du contenu sur votre site web.
```